src/tasks/installers/helm.ts

```diff
@@ -348,18 +348,9 @@ error: E_COMMAND_FAILED`)
     setOptions.push(`--set cheImage=${cheImage}`)
     setOptions.push(`--set che.disableProbes=${flags.debug}`)
 
-    if (flags['helm-patch-yaml']) {
-      // Read patch yaml. Has format the same as values.yaml
-      const patchTree: { [key: string]: any } = await this.kubeHelper.safeLoadFromYamlFile(flags['helm-patch-yaml'])
-      // Flat the yaml tree into key-value format
-      const patchProperties = this.prepareYamlPatch(patchTree)
-      // Add patch properties to the command
-      for (const property of patchProperties) {
-        setOptions.push(`--set ${property}`)
-      }
-    }
+    const patchFlags = flags['helm-patch-yaml'] ? '-f ' + flags['helm-patch-yaml'] : ''
 
-    let command = `helm upgrade --install che --force --namespace ${flags.chenamespace} ${setOptions.join(' ')} ${multiUserFlag} ${tlsFlag} ${destDir}`
+    let command = `helm upgrade --install che --force --namespace ${flags.chenamespace} ${setOptions.join(' ')} ${multiUserFlag} ${tlsFlag} ${patchFlags} ${destDir}`
```

Comment:
should `patchFlags` be at the very end ?

Reply:
```
Usage:
  helm upgrade [RELEASE] [CHART] [flags]
```
`flags` can be set at the end, or before (like `chectl` is doing).  Both will work.  In this case `${destDir}` is the `CHART` parameter.  If you want to follow helm's documented usage, you will need to shuffle more than that parameter.